apps/trading/components/sidebar/sidebar.spec.tsx

```diff
@@ -35,11 +37,15 @@ describe('Sidebar', () => {
   const renderComponent = () => {
     const user = userEvent.setup();
     const result = render(
-      <MemoryRouter initialEntries={[`/markets/${marketId}`]}>
-        <Routes>
-          <Route path="/markets/:marketId" element={<Sidebar />} />
-        </Routes>
-      </MemoryRouter>
+      <MockedProvider>
+        <MockedWalletProvider>
+          <MemoryRouter initialEntries={[`/markets/${marketId}`]}>
```

Comment:
There are no longer any Route elements within the sidebar so initialEntries can be removed